code/modules/error_handler/error_viewer.dm

```diff
@@ -112,7 +112,6 @@ var/global/datum/ErrorViewer/ErrorCache/error_cache = null
 
 	// Show the error to admins with debug messages turned on, but only if one
 	//  from the same source hasn't been shown too recently
-	// (Also, make sure config is initialized, or log_debug will runtime)
```

Comment:
This comment has vanished for some reason. This file shouldn't actually have *any* changes... did you commit the manual changes?